code.py

```diff
@@ -69,20 +69,25 @@ def nextDist(self, pose, line, index_bot):
 		
 		if d1 < d2:
 			self.pose[index_bot] = [x2, y2]
+			return d1
 		else:
 			self.pose[index_bot] = [x1, y1]
-		dist = min(d1,d2)
-		return dist
+			return d2
 
 
 	# Heuristics func in paper: Return distance/cost of a single input group
-	def groupDist(self, sol):
+	def groupDist(self, group):
 		index_line = 0
-		cost = 0
-		for index_bot in sol:
+		c = np.empty(len(self.pose))
```

Comment:
It's easier to use:
`c = [0 for i in range(len(self.pose))]`

Reply:
Oh, yeah. Will make the change